pkg/internal/checkup/checkup.go

```diff
@@ -47,6 +47,7 @@ type kubeVirtVMIClient interface {
 type Checkup struct {
 	client    kubeVirtVMIClient
 	namespace string
+	params    config.Config
```

Comment:
Could you please add an explanation in the commit message regarding why is it necessary?
For example: Cannot do my operations in new, and need to perform them in setup.

Please change the subject's prefix from mainflow to checkup.

Reply:
DONE